workspace/__lib__/xod-dev/w5500/ethernet-shield-device/patch.xodp

```diff
@@ -33,7 +33,7 @@
       "type": "xod/patch-nodes/not-implemented-in-xod"
     },
     {
-      "description": "MAC address of the shield",
+      "description": "The MAC address",
```

Comment:
A MAC address. Should be unique for a device. Often printed on the bottom or a package.